proposals/debug.log

```diff
@@ -0,0 +1,2 @@
+[1018/105456.328:ERROR:crash_report_database_win.cc(428)] unexpected header
+[1018/105924.215:ERROR:crash_report_database_win.cc(428)] unexpected header
```

Comment:
If this artifact comes from a setup specific to some developer(s), it might make more sense that they add it to global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore To me this log seems at least windows specific.